README.md

```diff
@@ -41,6 +41,8 @@ our [PicoSystem C++ API Cheatsheet](https://learn.pimoroni.com/article/picosyste
 
 ### Getting started
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/.../...)
```

Comment:
Will this link actually work? Does GitPod try to do something funky with the referrer?

Reply:
@Gadgetoid I don't think that'll work, but "https://gitpod.io/from-referrer" should (it's undocumented, but currently supported) - check link on the "Ready to code" badge on [GitHub](https://github.com/gitpod-io/gitpod)

Reply:
I thought `https` prevented referrer information from leaking, how is `from-referrer` working? Are my web chops just ten years out of date? :laughing:

I guess this should probably be changed to `from-referrer` to support opening forks/branches.